internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -135,6 +136,31 @@ func (r *AddOnDefinitionResource) Schema(ctx context.Context, req resource.Schem
 					},
 				},
 			},
+			"variable_options": schema.ListNestedAttribute{
+				Optional:    true,
+				Description: "List of variable options for the template",
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"name": &schema.StringAttribute{
+							Required:    true,
+							Description: "Variable name",
+						},
+						"variable_type": &schema.StringAttribute{
+							Required:    true,
+							Description: "Variable type",
+						},
+						"options": &schema.ListAttribute{
+							ElementType: types.StringType,
+							Required:    true,
+							Description: "List of options",
+						},
+						"user_editable": &schema.BoolAttribute{
+							Required:    true,
```

Comment:
This is `Optional` in [resource_waypoint_application_template.go](https://github.com/hashicorp/terraform-provider-hcp/blob/fc5c1b65e44e6395f78ed14c67b24147bc2c20c7/internal/provider/waypoint/resource_waypoint_application_template.go#L177). I believe it also needs to be `Computed: true`

Reply:
Updated in https://github.com/hashicorp/terraform-provider-hcp/pull/819/commits/81d17571516ab45f48be711b730cb91fa475031a (for both templates and add-on definitions).